### PR TITLE
The "I Swear I'm Not Going To Change View Anymore" Update + Tests

### PR DESCRIPTION
**Pull request by chrstinalin**
CHANGES

- Added missing window name, icon to ones that didn't have it (login view and search view)
- Fixed search view, removed playBar from search as requested ages ago (lol)
- Added icon for remove song
- Added shuffle button @rafaelgacesa 
- Minor ArtistView Fix (adjusted positioning)
- Added missing tests for my use cases.

### CODE REVIEW

**Review by SiqiZhang6 — APPROVED** [review]
Thank you so much for making the view nice, and the test looks great too!

**Review by rafaelgacesa — APPROVED** [review]
Nice tests but no javadocs (tbh idk if they are necessary but my IDE always bugs me about it).

Mostly view code, not much to comment on. We seriously have to stop changing the .csv files in main so much lol, it just increases the likelihood of some stupid NullPointer someone wastes their time on.

**Review by SiqiZhang6 — APPROVED** [review]
reapproving :D

File: src/main/java/database/SongDsData.java [hunk 1]
```diff
@@ -10,14 +10,6 @@
 public class SongDsData {
     private final Song song;
 
-    /**
-     * Constructor to be called from use case code to communicate with database
-     * @param song song object to encapsulate
-     */
-    public SongDsData(Song song){
```
Comment (rafaelgacesa) [inline src/main/java/database/SongDsData.java:17]: Wasn't this constructor necessary for uploading a song?  Or do you do it differently somehow

**Review by rafaelgacesa — APPROVED** [review]

File: src/main/java/database/SongDsData.java (hunk 1 above)
Comment (chrstinalin) [reply src/main/java/database/SongDsData.java:17]: It uses the other one -- and all the save/read stuff is in SongLibrary so it'd be counterintuitive to create a Song object inside database only to put it inside a SongDsData lol. Also Intellij prompted that it was never used 🤷

**Review by sarxhlwalker — APPROVED** [review]
looks good to me, thanks for this!

**Pull request merged**